src/resources/dart-3-migration.md

```diff
@@ -0,0 +1,512 @@
+---
+title: Dart 3 migration guide
+description: Tips for how to migrate existing Dart code to be compatible with Dart 3.
+---
+
+Dart 3 is a new major release of Dart, that brings new core capabilities like
+records, patterns, and classes. Alongside these new capabilities the release
+also contains a number of changes, that may break existing code. The present
+document is intended to be a migration guide that may be helpful in resolving
+these.
```

Comment:
```suggestion
Dart 3 is a major release that introduces new core capabilities to Dart: 
records, patterns, and class modifiers.

Alongside these new capabilities, Dart 3 contains a number of changes
that may break existing code. 
This migration guide explains the breaking changes in Dart 3
and how you can resolve them.
```

Reply:
I'll come back after my work gets published and add links on "records, patterns, and class modifiers"